core/include/usBundleContext.h

```diff
@@ -114,15 +166,15 @@ class US_Core_EXPORT BundleContext
    * @return A <code>Bundle</code> object or <code>nullptr</code> if the
    *         identifier does not match any previously installed bundle.
    */
-  std::shared_ptr<Bundle> GetBundle(long id) const;
+  Bundle GetBundle(long id) const;
 
   /**
-   * Get the bundle with the specified bundle name.
+   * Get the bundles with the specified bundle location.
    *
-   * @param name The name of the bundle to get.
-   * @return The requested \c Bundle or \c nullptr.
+   * @param name The location of the bundles to get.
+   * @return The requested {\c Bundle}s or an empty list.
```

Comment:
After seeing more instances of the use of "list" in the comments, I agree with you. The return type is clear from function signature.